backend/src/infrastructure/router/router.ts

```diff
@@ -21,6 +21,14 @@ export default class Router {
       );
     });
 
+    app.get('/books/:bookId', async (request, response, next) => {
```

Comment:
Estaría bien que pusieras el middleware de autenticación, para que solo usuarios con sesión válida puedan consultar info de libros. 
```suggestion
    app.get('/books/:bookId', middleware, async (request, response, next) => {
```

Lo dejo a tu consideración.